docs/examples/csfle-explicit_encryption.php

```diff
@@ -41,6 +41,8 @@
 
 /* Using the client configured without encryption, find the document and observe
  * that the field is not automatically decrypted. */
+
+/** @var object{encryptedField: Binary} $document */
```

Comment:
Looking at `psalm-baseline.xml`, this was not enough to enforce the type of `$document->encryptedField`?

Reply:
The error is regarding the return type of `decrypt`, which is mixed and can't be made any more specific:
> ERROR: MixedArgument - docs/examples/csfle-explicit_encryption.php:51:27 - Argument 2 of printf cannot be mixed, expecting float|int|string

The `@var` annotation here prevents a different Psalm error regarding accessing a potentially undefined property.